### PR TITLE
docs(RFC): external link glyph

### PR DESCRIPTION
**Pull request by juliusmarminge**
was brought up in another pr that we should mark external links.

not sure if I like it or not myself

CleanShot 2022-11-25 at 09 17 37@2x

### CODE REVIEW

**Review by nexxeln — APPROVED** [review]
I like it

**Comment by c-ehrlich** [comment]
I think it's alright, but:
1. i don't think this is the right icon/styling for it yet, these usually look something like this: (random google image search screenshot)
image
2. I think the reason this was brought up in first place is a different problem, which is that our color palette is missing contrast colors so it becomes kind of hard to parse the page. (but that doesn't mean we shouldn't add the glyph)

Compare Astro docs to us:


image

**Comment by nexxeln** [comment]
> I think it's alright, but:
> 
> 1. i don't think this is the right icon/styling for it yet, these usually look something like this:

Does it matter though? I really like how the arrow looks, very minimal

**Comment by juliusmarminge** [comment]
> i don't think this is the right icon/styling for it yet, these usually look something like this: (random google image search screenshot)

I don't think that icon is a valid unicode, I couldn't find any atleast. Not sure if you can put a proper svg

**Comment by asrvd** [comment]
github has something like this too, looks nice to me, maybe add a bit of gap between text and icon though
image

**Comment by juliusmarminge** [comment]
Added some padding but not the biggest fan that the underline "cuts off"

CleanShot 2022-11-27 at 11 39 53@2x

**Comment by nexxeln** [comment]
> Added some padding but not the biggest fan that the underline "cuts off"
> 
> CleanShot 2022-11-27 at 11 39 53@2x

it was fine without the padding imo

**Comment by juliusmarminge** [comment]
> > Added some padding but not the biggest fan that the underline "cuts off"
> > CleanShot 2022-11-27 at 11 39 53@2x
> 
> it was fine without the padding imo

Cool, I'll remove the padding then :)
